src/desktop/apps/consign/components/thank_you/index.js

```diff
@@ -1,21 +1,43 @@
 import Document from "../../../../components/main_layout/public/icons/consignments-doc.svg"
 import GreenCheck from "../../../../components/main_layout/public/icons/green-check.svg"
 import PropTypes from "prop-types"
-import React from "react"
+import React, { useEffect } from "react"
 import Select from "../../../../components/main_layout/public/icons/consignments-select.svg"
 import SpeechBubble from "../../../../components/main_layout/public/icons/consignments-speech-bubble.svg"
 import block from "bem-cn-lite"
 import { get } from "lodash"
 import { connect } from "react-redux"
+import request from "superagent"
+import { AnalyticsSchema } from "@artsy/reaction/dist/Artsy"
 
-function ThankYou({ isMobile, submission, processingImages, uploadedImages }) {
+function ThankYou({
+  contextPath,
+  isMobile,
+  submission,
+  processingImages,
+  uploadedImages,
+}) {
   const b = block("consignments-submission-thank-you")
   const uploadedImageSrc = get(uploadedImages, "0.src")
   const submissionImage =
     uploadedImageSrc && processingImages.length === 0
       ? uploadedImageSrc
       : "/images/missing_image.png"
 
+  /**
+   * To get around the possibility that an AdBlocker might block a successful
+   * submission event send the event via API request.
+   */
+  useEffect(() => {
+    request
+      .post(
+        `/consign/track-submission-success?contextPath=${contextPath}&subject=${AnalyticsSchema.Subject.SubmitForReview}`
+      )
```

Comment:
@louislecluse - I'm noticing that we also use the subject `submit for review` in the main home button: 

<img width="456" alt="Screen Shot 2020-04-18 at 11 40 21 AM" src="https://user-images.githubusercontent.com/236943/79662393-6be5e880-8169-11ea-8c61-c37324549525.png">

But the user has not yet submitted a a consignment for review, they've only started the process. Where here, at this point in the code, they've submitted successfully. 

Should we create a new subject for that main button so that there's a clear distinction between starting and ending the process of submission? 

Also, should we be sending `submissionID` along to the server-side event?

Reply:
Not sure I get that one. Is it for the end of the submission flow?

Reply:
Ok I understand now, see [this](https://github.com/artsy/force/pull/5439#issuecomment-616606696) for comment on the event itself.
Otherwise, for the naming question, can't we call the event for instance: `submitted consignment`?